docs/usage/executing-commands.md

```diff
@@ -0,0 +1,61 @@
+---
+id: executing-kubectl-commands
+title: "Executing kubectl commands"
+sidebar_position: 2
+---
+
+# Executing kubectl commands
+
+Botkube allows you to execute kubectl commands on your Kubernetes cluster. By default, kubectl command execution is disabled. To enable this feature, set `executors.<configuration-name>.kubectl.enabled: true` in [Executor configuration](../configuration/executor).
+
+As suggested in help message, to execute kubectl commands, send message in following format in the channel where Botkube is already added or as a direct message to Botkube.
+
+```
+@Botkube <kubectl command with or without `kubectl` prefix> [--cluster-name <cluster_name>]
+```
+
+:::info
+You can also prefix your commands with `kubectl` , `kc` or `k`.
+:::
+
+:::caution
+In future, one of the kubectl prefix (`kubectl` , `kc` or `k`) will be required.
+:::
+
+### Checking allowed commands
+
+While deploying Botkube controller, you can specify which kubectl commands you want to allow Botkube to execute through <a href="../configuration">configuration</a>.
+
+To check which commands are allowed for users to execute through Botkube, run **@Botkube commands list**
+
+![commands_list](assets/commands_list.png)
+
+### Run Kubectl commands
+
+Botkube configuration allows you to give execution of kubectl commands
+
+Run **@Botkube < kubectl command without kubectl prefix >** to get kubectl response from the cluster configured with the channel.
```

Comment:
```suggestion
Run **@Botkube  <kubectl command with or without `kubectl` prefix>** to get kubectl response from the cluster configured with the channel.
```

BTW we already have that in the overview, maybe we can remove it from here 🤔

Reply:
Good catch, I didn't edit it, but will fix it 👍